web/pandas/getting_started.md

```diff
@@ -1,5 +1,16 @@
 # Getting started
 
+## Try it online
+
+You can try `pandas` online with the following interactive shell
+without installing anything on your computer:
+
+<iframe
+  src="https://jtpio.github.io/pandas-repl/repl/index.html?toolbar=1&kernel=python&code=import pandas as pd"
```

Comment:
This will have to be updated after the repo is transferred to `pandas-dev` organization: https://github.com/jtpio/pandas-repl

Reply:
Usually it's simpler to keep the content in a separate repo because the JupyterLite website is then deployed to GitHub Pages automatically when new changes are merged.

But if there is already a CI workflow deploying the pandas website on push, then this could also be integrated there.

Reply:
> But if there is already a CI workflow deploying the pandas website on push, then this could also be integrated there.

Looks like this might be relevant: 

https://github.com/pandas-dev/pandas/blob/3364f9af4c5e3ea1d5585217d8c97cfe6631d8e1/.github/workflows/docbuild-and-upload.yml#L60-L62

Reply:
In that case the content currently in https://github.com/jtpio/pandas-repl could be placed under `web/repl` for instance.

And then there could be an additional step running on CI to build the JupyterLite assets with `jupyter lite build` to `web/build/lite`, which would then be copied by the existing `rsync` command?

Reply:
+1 with this approach. I'd keep the assets for the terminal in `web/interactive_terminal` or similar name.I think it's clearer. When building, `web/build/lite` or whatever it's standard is fine. If you copy files to that directory, the last steps of the website workflow should already take care of deploying it.

Also fine to keep things simple here, and create a DataFrame from the constructor first. We can discuss in a separate issue PR about the exact data and examples to use. Probably better to not mix the technical work with JupyterLite and the content of the terminal in the same discussion I think.

An example as simple as next would work for me:
```python
df = pd.DataFrame({'num_legs': [2, 4], 'num_wings': [2, 0]},
                  index=['falcon', 'dog'])
```

Reply:
Sound good :+1: